client/src/partials/patients/groups/groups.html

```diff
@@ -1,151 +1,167 @@
+            </thead>
+            <tbody>
+              <tr ng-repeat="group in patientGroupCtrl.groups">
+                <td>{{ group.name}}</td>
+                <td>{{ group.priceListLable }}</td>
+                <td>{{ group.note }}</td>
+                <td>
+                  <a class="action" ng-click="patientGroupCtrl.update(group.uuid)"><i class="glyphicon glyphicon-pencil"></i></a>
+                </td>              
+              </tr>
+              <tr ng-if="patientGroupCtrl.loading" class="text-center">
+                <td colspan="4"><loading-indicator></loading-indicator></td>
+              </tr>
+            </tbody>
+          </table>
         </div>
-        <table class="table table-condensed">
-          <thead>
-            <tr>
-              <th>{{ "PATIENT_GRP.PATIENT_GROUP" | translate }}</th>
-              <th>{{ "PATIENT_GRP.PRICE_LIST" | translate }}</th>
-              <th>{{ "PATIENT_GRP.SUBSIDY" | translate }}</th>
-              <th colspan="3" width="10%">Actions</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr ng-class="{'selected-primary' : session.selected.uuid === grp.uuid}" ng-repeat="grp in group.data  | orderBy:'name'">
-              <td>{{ grp.name }}</td>
-              <td>{{ list.get(grp.price_list_uuid).title }}</td>
-              <td>{{ subsidies.get(grp.subsidy_uuid).text }}</td>
-              <td><a ng-if="grp.price_list_uuid" title="{{ 'PATIENT_GRP.PRICE_LIST' | translate }}" class="action" ng-click="showReport(grp)"><span class="glyphicon glyphicon-eye-open"></span></a></td>
-              <td><a class="action" ng-click="edit(grp)"><i class="glyphicon glyphicon-pencil"></i></a></td>
-              <td><a class="action" ng-click="remove(grp)"><i class="glyphicon glyphicon-trash uif-red"></i></a></td>
-            </tr>
-          </tbody>
-        </table>
       </div>
-    </div>
 
-    <div class="col-xs-6" ng-switch="action">
-
-      <div ng-switch-default>
-        <div class="alert alert-info">
-          <h4>{{ "PATIENT_GRP.TITLE" | translate }}</h4>
-          <p>{{ "PATIENT_GRP.HELP_TXT_1" | translate }}.</p>
-        </div>
-      </div>
-
-      <div ng-switch-when="register">
-        <form class="form-horizontal" name="add" novalidate>
-          <fieldset>
-            <legend>{{ "PATIENT_GRP.REGISTER" | translate }}</legend>
-
-            <div class="form-group required">
-              <label class="col-xs-2 control-label">{{ "COLUMNS.NAME" | translate }}</label>
-              <div class="col-xs-10">
-                <input class="form-bhima" ng-model="register.name" required>
+      <div class="col-md-6" ng-switch="patientGroupCtrl.view">
+
+        <div ng-switch-default>
+          <div class="alert alert-info" id="default">
+            <h4>{{ "PATIENT_GROUP.TITLE" | translate }}</h4>
+            <p>{{ "PRICE_LIST.DESCRIPTION1" | translate }}</p>
+            <ul style="list-style-type: none; padding-left: 15px;">
```

Comment:
Bootstrap has a class ```list-unstyled```. It's much more maintainable to use classes vs. in-line styles where possible - this is often good for quick development it can just be very quickly re-factored.